packages/yew-router-macro/tests/routable_derive/valid-pass.rs

```diff
@@ -6,8 +6,8 @@ enum Routes {
     One,
     #[at("/two/:id")]
     Two { id: u32 },
-    #[at("/:a/:b")]
-    Three { a: u32, b: u32 },
+    #[at("/:a/:b/*rest")]
+    Three { a: u32, b: u32, rest: ::std::string::String },
```

Comment:
I dislike this as it opts out of controlling your routes as enums.

Reply:
It doesn't have to. The rest parameter taken from here can then be passed to `Switch`, which can match on it according to the `Routable`